e2e/pages/events/createOrderEvent.page.js

```diff
@@ -1,15 +1,16 @@
 const {I} = inject();
 const judgeAndLegalAdvisor = require('../../fragments/judgeAndLegalAdvisor');
+const orders = require('../../fixtures/orders.js');
 
 module.exports = {
   fields: {
-    orderTitle: '#c21Order_orderTitle',
-    orderDetails: '#c21Order_orderDetails',
+    orderTitle: '#order_orderTitle',
+    orderDetails: '#order_orderDetails',
   },
 
-  enterOrder() {
-    I.fillField(this.fields.orderTitle, 'Example Title');
-    I.fillField(this.fields.orderDetails, 'Example order details here - Lorem ipsum dolor sit amet...');
+  enterC21OrderDetails() {
```

Comment:
Why the name change here?

Reply:
in preparation for future orders and just to be a bit more specific - the other orders don't have a title or details that the user inputs, only C21 does